### PR TITLE
chore: [M3-6557] - Improve Premium plans Tab copy

### PR DESCRIPTION
**Pull request by cpathipa**
## Description 📝
**Updated copy of Premium plans tab**

| Before  | After   |
| ------- | ------- |
| image | image |

## How to test 🧪
1. **Navigate to Plans table validate copy under Premium plans tab**

### CODE REVIEW

File: packages/manager/src/features/Linodes/LinodesCreate/SelectPlanPanel/utils.ts [hunk 1]
```diff
@@ -155,7 +155,7 @@ export const planTabInfoContent = {
   },
   premium: {
     typography:
-      'Premium CPU instances guarantee a minimum processor model, AMD Epyc\u2122 7713 or higher, to ensure consistent high performance for more demanding workloads.',
+      'Premium CPU instances guarantee a minimum processor generation, of AMD EPYC(TM) 7713 or higher, to ensure consistent high performance for more demanding workloads.',
```
Comment (hana-linode) [inline packages/manager/src/features/Linodes/LinodesCreate/SelectPlanPanel/utils.ts:158]: I think the sentence reads better without the comma after "generation"

File: packages/manager/src/features/Linodes/LinodesCreate/SelectPlanPanel/utils.ts [hunk 2]
```diff
@@ -155,7 +155,7 @@ export const planTabInfoContent = {
   },
   premium: {
     typography:
-      'Premium CPU instances guarantee a minimum processor model, AMD Epyc\u2122 7713 or higher, to ensure consistent high performance for more demanding workloads.',
+      'Premium CPU instances guarantee a minimum processor generation of AMD EPYC\u2122 Milan or newer, to ensure consistent high performance for more demanding workloads.',
```
Comment (mjac0bs) [inline packages/manager/src/features/Linodes/LinodesCreate/SelectPlanPanel/utils.ts:158]:
```suggestion
      'Premium CPU instances guarantee a minimum processor generation of AMD EPYC\u2122 Milan or newer to ensure consistent high performance for more demanding workloads.',
```

One more tweak here, I think -- we shouldn't need a comma after "to" and before "ensure" either; it breaks the flow of the sentence.

**Review by mjac0bs — APPROVED** [review]
Copy matches the internal ticket now - just one final punctuation change requested. Thanks for the copy updates!

**Review by hana-linode — APPROVED** [review]

File: packages/manager/src/features/Linodes/LinodesCreate/SelectPlanPanel/utils.ts (hunk 2 above)
Comment (bnussman-akamai) [reply packages/manager/src/features/Linodes/LinodesCreate/SelectPlanPanel/utils.ts:158]: I agree

**Review by bnussman-akamai — APPROVED** [review]

**Pull request merged**